src/olympia/stats/utils.py

```diff
@@ -111,3 +115,76 @@ def get_addons_and_average_daily_users_from_bigquery():
         (row['addon_id'], row['count'])
         for row in rows if row['addon_id'] and row['count']
     ]
+
+
+def get_averages_by_addon_from_bigquery(today):
+    """This function is used to compute the 'hotness' score of each add-on (see
+    also `deliver_hotness()` cron task)."""
+    client = bigquery.Client.from_service_account_json(
+        settings.GOOGLE_APPLICATION_CREDENTIALS
+    )
+
+    one_week_date = today - timedelta(days=7)
+    four_weeks_date = today - timedelta(days=28)
+
+    fully_qualified_table_name = '.'.join(
```

Comment:
This is defined 3 times in this file; can we abstract it to a constant or function ?